### PR TITLE
feat/inheritance edit encrypted message

### PR DESCRIPTION
**Pull request by AryanCypherock**

### CODE REVIEW

File: packages/cysync-core/src/actions/dialog/index.ts [hunk 1]
```diff
@@ -103,3 +103,6 @@ export const openInheritanceEditReminderTimeDialog = () =>
 
 export const openEditUserDetailsDialog = () =>
   openDialog({ name: 'inheritanceEditUserDetails', data: undefined });
+
+export const openEditEncryptedMessageDialog = () =>
```
Comment (irshadjsr21) [inline packages/cysync-core/src/actions/dialog/index.ts]:
```suggestion
export const openInheritanceEditEncryptedMessageDialog = () =>
```

File: packages/cysync-core/src/actions/dialog/index.ts [hunk 2]
```diff
@@ -103,3 +103,6 @@ export const openInheritanceEditReminderTimeDialog = () =>
 
 export const openEditUserDetailsDialog = () =>
```
Comment (irshadjsr21) [inline packages/cysync-core/src/actions/dialog/index.ts]:
```suggestion
export const openInheritanceEditUserDetailsDialog = () =>
```

File: packages/cysync-core/src/dialogs/Inheritance/EditEncryptedMessage/Dialogs/Decryption.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      <LoaderDialog
+        title={strings.decryption.title}
+        subtext={strings.decryption.subTitle}
+      />
+    );
+  }
+
+  return (
+    <Layout>
+      <ReactPlayer
```
Comment (irshadjsr21) [inline packages/cysync-core/src/dialogs/Inheritance/EditEncryptedMessage/Dialogs/Decryption.tsx]: Try to use `Video` component
```suggestion
      <Video
```

File: packages/cysync-core/src/dialogs/index.tsx [hunk 4]
```diff
@@ -38,6 +38,7 @@ import { SignMessageDialog } from './SignMessage';
 import { WalletActionsDialogBox } from './WalletActions';
 import { WalletConnectDialog } from './WalletConnect';
 import { WalletSyncError } from './WalletSyncError';
+import { InheritanceEditEncryptedMessageDialog } from './Inheritance/EditEncryptedMessage';
```
Comment (irshadjsr21) [inline packages/cysync-core/src/dialogs/index.tsx]: Import from `./Inheritance`

**Review by irshadjsr21 — CHANGES_REQUESTED** [review]

File: packages/cysync-core/src/dialogs/Inheritance/EditEncryptedMessage/Dialogs/EditMessage.tsx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    lang.strings.dialogs.inheritanceEditEncryptedMessage.editMessage;
+  const { onClose, onNext } = useInheritanceEditEncryptedMessageDialog();
+
+  return (
+    <Layout
+      footerComponent={
+        <>
+          <Button onClick={() => onClose()}>
+            <LangDisplay text={lang.strings.buttons.exitWithoutSaving} />
+          </Button>
```
Comment (TejasvOnly) [inline packages/cysync-core/src/dialogs/Inheritance/EditEncryptedMessage/Dialogs/EditMessage.tsx]: Requires secondary variant here

**Review by TejasvOnly — CHANGES_REQUESTED** [review]

**Review by TejasvOnly — APPROVED** [review]

**Pull request merged**